nixos/platform/collect-garbage.nix

```diff
@@ -6,7 +6,7 @@
 with builtins;
 
 let
-  cfg = config.flyingcircus;
+  cfg = config.flyingcircus.agent;
   fclib = config.fclib;
   log = "/var/log/fc-collect-garbage.log";
```

Comment:
this is actually a stamp file which is read by sensu

Reply:
stamp file is back now, but it's explicitly named as such instead of calling it a log file. Also, it's placed under /var/lib/fc-collect-garbage now to make clear that it belongs to the fc-collect-garbage service